### DISCUSSION TITLE
Make member functions const when appropriate.

### DISCUSSION THREAD

**Discussion by IgorJamder** [uNetworking/uWebSockets#1239] [General]
Please consider making appropriate 'uWS::HttpRequest' member function const, e.g.:
getMethod() const
getUrl() const
getQuery() const
etc.

(also adding cbegin(), cend() would be great :o)  - to simplify the iteration over the headers using std:: algos, etc.

**Comment by ghost** [comment]
Can you PR the const fixes?
